### ISSUE TITLE
Android not compiling on RN v0.32.0-rc.0 (TouchEvent.obtain)

### ISSUE THREAD

**Issue by alvinlao** [react-native-community/react-native-svg#118]
# Problem
This library is not compiling on Android with React Native version `0.32.0-rc.0`.

# Error
This is the compilation error I got:
```js
node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/RNSVGSvgView.java:114: error: method obtain in class TouchEvent cannot be a
pplied to given types;
            TouchEvent.obtain(
                      ^
  required: int,TouchEventType,MotionEvent,float,float,TouchEventCoalescingKeyHelper
  found: int,long,TouchEventType,MotionEvent,float,float,TouchEventCoalescingKeyHelper
  reason: actual and formal argument lists differ in length
```

# Cause
`TouchEvent.obtain`'s signature changed in [this commit](https://github.com/facebook/react-native/commit/da063e3d55a4fcd617ac496d62aa051343f35b56).

# Solution
The fix should be to remove `SystemClock.nanoTime()` from line 116 in `RNSVGSvgView.java`.

```java
    private void dispatch(MotionEvent ev, TouchEventType type) {
        mEventDispatcher.dispatchEvent(
            TouchEvent.obtain(
                mTargetTag,
                SystemClock.nanoTime(),  // <------- Remove this line
                type,
                ev,
                ev.getX(),
                ev.getX(),
                mTouchEventCoalescingKeyHelper));
    }
```

Thanks!

**Comment by magicismight** [comment]
Thanks for reporting this.
I\`ll fix this after 0.32.0 is released.

**Comment by fauno** [comment]
thanks for the fix! :)
